config/jobs/kubernetes-sigs/cluster-api-provider-openstack/cluster-api-provider-openstack-presubmits.yaml

```diff
@@ -17,7 +17,7 @@ presubmits:
         # docker-in-docker needs privileged mode
         securityContext:
           privileged: true
-        image: gcr.io/k8s-testimages/kubekins-e2e:v20181005-fd9cfb8b0-1.10
+        image: gcr.io/k8s-testimages/kubekins-e2e:v20181010-2672d9494-master
```

Comment:
it's a dind build job so... go version doesn't really matter... use master tag instead.